framework/calico_framework.py

```diff
@@ -0,0 +1,661 @@
+#!/usr/bin/env python
+
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import sys
+import subprocess
+import re
+import json
+import time
+import threading
+import math
+from itertools import izip_longest
+from random import randint
+import mesos.interface
+from mesos.interface import mesos_pb2
+import mesos.native
+from calico_utils import _setup_logging
+from framework.tasks import SleepTask, PingTask, NetcatListenTask, NetcatSendTask
```

Comment:
Can we please have a naming structure that avoids `/framework/framework/` in the path?